seoulstice/le_pivot
# refactored if else logic by removing exccess conditionals and || stat…

…ements

#### What does  this PR do?
ID 154940134 on Pivotal Tracker, refactored logic in views by removing excess if statement and removing || conditionals within the logic. Logic is now a simple if/elsif/else statement. Tests coverage is the same as it was before (2 failing tests)
#### Where should the reviewer start?
Look at admin dashboard show line 49
#### How should this be manually tested?
Bundle Exec Rspec should show 2 failing tests, as it was before. Also  manually check this with running through code on local (Admin login is mary@example.com and password is password)
#### Any background context you want to provide?
Nope
#### What are the relevant story numbers?
154940134
#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran?
  - Do Environment Variables need to be set?
  - Any other deploy steps?

None
